plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php

```diff
@@ -211,7 +212,7 @@ public static function get_all() {
 				'recommendation_priority' => 1,
 			),
 			array(
-				'id'                      => 'woocommerce_payments:non-us',
+				'id'                      => 'woocommerce_payments:without-in-person-payments',
```

Comment:
I changed this because we now need to include CA in personal payments. If it's not safe, I think we can add additional WCPay entries and set up plugin version rules, but the downside is that we may have many wcapy items.

Reply:
I'm not entirely sure if we have other uses for this id. My search landed on some discussion on slotfill using this id p1645113034234109/1645107773.216159-slack-C9K5T0XSP

Reply:
> My search landed on some discussion on slotfill using this id p1645113034234109/1645107773.216159-slack-C9K5T0XSP

Thanks, good find, @ilyasfoo!

I was not able to find the related code in WCPay. 💭  Hey @ismaeldcom 👋  Could you please confirm if it's okay to change the ids (`woocommerce_payments:non-us` and `woocommerce_payments:us`)? Thanks!

Reply:
Sure! No issue changing them for the onboarding optimization project. We end up using an alternative approach due to WCPay use of older dependencies.

Reply:
@ismaeldcom Thanks for the confirmation!